src/apps/ubeswap/positions.ts

```diff
@@ -138,14 +141,17 @@ async function getPoolPositionDefinitions(
 ): Promise<PositionDefinition[]> {
   // Get the pairs from Ubeswap via The Graph
   const { data } = await got
-    .post('https://api.thegraph.com/subgraphs/name/ubeswap/ubeswap', {
-      json: {
-        query: PAIRS_QUERY,
-        variables: {
-          address: address.toLowerCase(),
+    .post(
+      'https://gateway-arbitrum.network.thegraph.com/api/3f1b45f0fd92b4f414a3158b0381f482/subgraphs/id/JWDRLCwj4H945xEkbB6eocBSZcYnibqcJPJ8h9davFi',
```

Comment:
I found on the network tab of https://info.ubeswap.org/ they were using this graph now. Not sure how long it's going to last though, Ubeswap seems to be kind of inactive, even with their V3 launch.

Reply:
Thanks, this is good for now! Weird it says `gateway-arbitrum` though 👀